docs/conf/portland/2025/cfp.rst

```diff
@@ -194,14 +185,45 @@ Timeline overview
 Example proposal
 ----------------
 
-If you'd like some guidance on how to create a talk proposal, take a look at our :doc:`Example proposal <example-proposal>`.
+Below is additional information and ideas to better complete specific prompts in the proposal. 
```

Comment:
```suggestion
Take a look at our example proposal, with additional guidance on the proposal format:
```

I think this works a lot better as a standalone linked page, as it highlights the format better. I can't easily tell if there are diffs there either.